gaurav/src/pages/ForgotPage.jsx

```diff
@@ -0,0 +1,66 @@
+import Button from "../components/ui/Button";
+import Card from "../components/ui/Card";
+import Input from "../components/ui/Input";
+import logo from "../assets/logo.png";
+import { Link } from "react-router-dom";
+
+export const // eslint-disable-next-line react/prop-types
+  ForgotPage = ({ handleSubmit, errorName, value, change }) => {
+    return (
+      <div className="h-screen w-full">
+        <div className="bg-login-image bg-cover h-screen w-full">
+          <a href="https://fr.stg.shipglobal.in/" className="">
+            <div className="flex justify-center items-start p-4 md:justify-start lg:py-8 lg:mx-16">
+              <img src={logo} style={{ height: 48 }} />
+            </div>
+          </a>
+          <div className="flex justify-center items-center m-4 mt-32 md:mt-[118px]">
```

Comment:
don't use dimensions which are not available in tailwind css. You can use the nearest possible value.